docs/assets/stylesheets/components/_documentation_tabs.scss

```diff
@@ -0,0 +1,284 @@
+.app-navigation {
```

Comment:
still not wild about this classname, but I can live with it.

However I think some of these customisations may need to also include the `.js-enabled` class, or some extra work needs to happen for the no-js version as it currently looks like this:

![image](https://github.com/user-attachments/assets/6b75eb51-140a-48a2-b64a-c5c05211deaa)

The gov.uk tabs look like this (without JS):
![image](https://github.com/user-attachments/assets/6968d90b-5fc5-4efb-bbc8-bf00497b9e34)

I think I'd err towards having it look as per gov.uk tabs without js and then enhance to what we have now. Might be as simple as `.js-enabled .app-navigation`